pkg/endpoints/dashboard.go

```diff
@@ -241,9 +241,9 @@ func GetTriggersVersion(r Resource, isOpenShift bool) string {
 }
 
 // Check whether Tekton Triggers is installed
-func IsTriggersInstalled(r Resource, isOpenShift bool) bool {
+func IsTriggersInstalled(r Resource, installedNamespace string, isOpenShift bool) bool {
 	isTriggersInstalled := false
-	namespacesToCheck := []string{"tekton-pipelines"}
+	namespacesToCheck := []string{installedNamespace}
```

Comment:
Same comment as above for the func `GetTriggersVersion` about triggers install namespace